### PR TITLE
Allow bare `ocm login` to suggest the token page without extra noise

### PR DESCRIPTION
**Pull request by cben**
@jhernand please review.
This guides users towards using --token from https://cloud.redhat.com/openshift/token.
That page is not yet advertised in UI but I think we should, as that's the sane ways for customers to access the API?

```sh
$ ocm login
In order to log in it is mandatory to use '--token', '--user' and '--password', or '--client-id' and '--client-secret'.
You can obtain a token at: https://cloud.redhat.com/openshift/token .
See 'ocm login --help' for full help.
$ echo $?
1
```

Works with some flags too, as long as you didn't provide crendentials:

```sh
$ ocm login --url=staging
In order to log in it is mandatory to use '--token', '--user' and '--password', or '--client-id' and '--client-secret'.
You can obtain a token at: https://cloud.redhat.com/openshift/token .
See 'ocm login --help' for full help.
$ echo $?
1
```

(In my testing, the same token is valid for logging into production, staging, and integration.)

Full help:

```sh
$ ocm login --help
Log in, saving the credentials to the configuration file.
The recommend way is using '--token', which you can obtain at: https://cloud.redhat.com/openshift/token

Usage:
  ocm login [flags]

Flags:
      --.....
```

(and exactly same for `ocm help login`)

- I'd like to later improve the token page similarly to openshift console's — link it from (?) icon -> Command Line Tools, and reword it to advertise this CLI and suggest `ocm login --token`, only later discussing direct low-level usage.

https://jira.coreos.com/browse/SDA-1609

### CODE REVIEW

File: cmd/ocm/login/cmd.go [hunk 1]
```diff
@@ -144,8 +147,16 @@ func run(cmd *cobra.Command, argv []string) error {
...
 	haveToken := args.token != ""
 	if !havePassword && !haveSecret && !haveToken {
-		return fmt.Errorf("In order to log in it is mandatory to use '--token', '--user' and " +
-			"'--password', or '--client-id' and '--client-secret'.")
+		// Allow bare `ocm login` to suggest the token page without extra noise.
+		fmt.Fprintf(
+			os.Stderr,
+			"In order to log in it is mandatory to use '--token', '--user' and "+
+				"'--password', or '--client-id' and '--client-secret'.\n"+
+				"You can obtain a token at: %s .\n"+
```
Comment (cben) [inline cmd/ocm/login/cmd.go]: Space before period is awkward but deliberate.
Not all terminal allow direct clicking of URLs, not all users know how to use that (e.g. Ctrl+click in gnome-terminal, or right click menu). The space makes it easier to grab the URL with regular mouse selection or double-click word selection.

Could trop the period altogether, but with the 2 other sentences ending in a period felt that would look awkward too.

File: cmd/ocm/login/cmd.go [hunk 2]
```diff
@@ -144,8 +147,16 @@ func run(cmd *cobra.Command, argv []string) error {
...
+				"See 'ocm login --help' for full help.\n",
+			uiTokenPage,
+		)
+		return nil
```
Comment (jhernand) [inline cmd/ocm/login/cmd.go]: This should `os.Exit(1)` otherwise scripts that use the tool will assume it succeeded in this case when it actually failed.

Comment (cben) [reply cmd/ocm/login/cmd.go]: Thanks, done.
